content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+
+<video src="./div.mp4" width="100%" height="auto" preload="auto" title="Voiceover reading of the content above"  controls></video>
+
+Likewise, search engine crawlers won't know that this is a list. If you're only using `div` tags as far as Google's concerned, you have no lists, no headings, nothing. This makes the page significantly less engaging and therefore rank more poorly.
+
+Let's compare that to using the correct HTML tags for a list.
+
+```html
+<ul aria-label="List of fruits">
+    <li>Orange</li>
+    <li>Banana</li>
+    <li>Grapefruit</li>
+</ul>
+```
+
+<video src="./semantic.mp4" width="100%" height="auto" preload="auto" title="Voiceover reading of the content above" controls></video>
+
+As you can hear, this screen-reader is now able to read out that it's a list. It makes navigation of that list easier for those users by allowing them to quickly skip to the next list item and hear the index of an item in the list.
+
+Not only will this enhance the experience of your screen-reader using users, but because search engine crawlers rely on HTML tags to inform what's what, your site will rank better in search engine queries as well! This is a massive boon to your site's SEO score.
+
+# Understand `aria-` properties {#aria}
+
+In our previous example, we used an HTML attribute [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) on our `ul`. [ARIA is collection of HTML attributes that allow you to enhance the accessibility in applications](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA). That said, _**it is highly encouraged to use the suggested HTML tags instead of `aria` attributes whenever possible**_. Think of `aria` as a complex low level API that can enhance your experience when done properly, but drastically harm user experience when unproperly utilized.
+
+A super small small subsection of `aria-` attributes includes:
+
+- [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) - Associate the element with another element's text as the label 
+- `aria-expanded` - A Boolean value meant to communicate when a dropdown is expanded
+- `aria-valuemin` - The minimum allowed value in a numerical input
+- `aria-valuemax` - The maximum allowed value of a numerical input
+
+Additional to `aria` props, [the `role` property](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#roles) acts to have the browser see and read an element as a different one. Again, this is a highly advanced (and often incorrectly deployed) API for complex apps. To learn more, [read through Mozilla's ARIA basics article.](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics)
+
+# Contrast is Cool {#contrast}
```

Comment:
I just noticed this. You should only have one h1 element for the title of the page. You should increase all non-frontmatter headings by one or write a transformer that does the same thing at build time, and errors on h6 -> h7.

Reply:
Yeah, this needs to be a larger change on UU. I want to make that a separate PR from this one, as I think we need to rework all of our other articles to work with this as well (some use H2, some use H1, let's standardize on `#` -> H2)